src/incubator/expandableOverlay/expandableOverlay.api.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
This is also an infra component, we used internally and it serves other components 
but IMO we shouldn't expose it because then we'll have to worry about breaking it